### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Vasyl-Poremchuk**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
```
Comment (HStakhiv) [inline app/main.py:7]: Probably better to use: "self.__class__.people[name]"

**Review by HStakhiv — APPROVED** [review]
Ok. But.....

**Review by HStakhiv — CHANGES_REQUESTED** [review]
Sorry, I changed my mind. Make the changes I wrote above :)

**Review by HStakhiv — APPROVED** [review]
Nice!

File: app/main.py (hunk 1 above)
Comment (Vasyl-Poremchuk) [reply app/main.py]: Done!

**Review by HStakhiv — APPROVED** [review]

**Review by Y-Havryliv — APPROVED** [review]
GJ!

**Pull request closed without merging**